ui/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessService.scala

```diff
@@ -235,6 +236,12 @@ class DBProcessService(managerActor: ActorRef,
         Future(Left(ProcessNotFoundError(processIdWithName.id.value.toString)))
     }
 
+  override def getUserProcesses[PS: ProcessShapeFetchStrategy](user: LoggedUser): Future[List[BaseProcessDetails[PS]]] = {
+    val userCategories = processCategoryService.getUserCategories(user)
+    val shapeStrategy = implicitly[ProcessShapeFetchStrategy[PS]]
```

Comment:
no quite sure why it's better than `implicit val luser: LoggedUser: user` ;)

Reply:
It looks more scala pro! :) But to be honest I don't say it's better but I think it looks better.